### PR TITLE
Cosmetic improvements (listed below) + community functionality

### PR DESCRIPTION
**Pull request by mrzengel**
Community Functionality: clicking on a community now let's you search within the community until the X is pressed
Cosmetic improvements:
- no more weird horizontal scrolling when record info is opened
- author list now mimics Zenodo
- Title is now marked with "Title" and is a link to the record on zenodo
- Files are only listed as file names and clicking on them downloads the files

### CODE REVIEW

**Comment by mrzengel** [comment]
Note: I didn't make this PR close issue #44 because there might be more cosmetic stuff we want changed/to add.

File: src/components/SearchPanel.tsx [hunk 1]
```diff
@@ -239,23 +383,31 @@ const SearchWidget: React.FC = () => {
...
                                                                 </div>
                                                             )}
-                                                            {recordInfo.filelist && (
+                                                            {recordInfo.filelist.length > 0 && (
                                                                 <div>
                                                                     <p><strong>Files:</strong></p>
                                                                     <ul>
                                                                         {recordInfo.filelist.map((file: string, index: number) => (
-                                                                            <li key={index}>{file}</li>
+                                                                            <li key={index}><a href={cleanUrl(file)} target='_blank' rel='noopener noreferrer'>{getFileNameFromUrl(file)}</a></li>
```
Comment (Soap2G) [inline src/components/SearchPanel.tsx:410]: This downloads files on the PC of the user, correct? 
Not yet in the jupyterlab session?

Comment (mrzengel) [reply src/components/SearchPanel.tsx:410]: Yes it was just a “first draft” essentially for that feature

Comment (mrzengel) [reply src/components/SearchPanel.tsx:410]: I can remove that functionality from that pull request if y’all want

Comment (Soap2G) [reply src/components/SearchPanel.tsx:410]: Forgot about #43, apologies
lgtm.

**Review by Soap2G — APPROVED** [review]

**Pull request merged**